tools/gui/README.md

```diff
@@ -0,0 +1,85 @@
+# MMEditing Viewer
+
+## Introduction
+
+**MMEditing Viewer** is a qualitative comparison tools to facilitate your research.
+
+## Major features
+
+- **Patch-based comparison**
+  - Crop a patch on multiple images to compare
+  - Batch comparison
+  - Flexible settings on number of columns and size of images.
+  - Save your comparison result
+- **Before/After slider comparison**
+  - Support both videos and images comparison
+  - Record and save comparison results as a video clip
+
```

Comment:
You may need add a section about environment/prerequirements/platform to introduce:
supported python version, etc.